Madness.xcworkspace/xcshareddata/Madness.xcscmblueprint

```diff
@@ -7,14 +7,16 @@
     "9A760A1EBAC1979D46EF016B1BE84D98E53C16FF" : 0,
     "7FCD68358D0D5A67F26F2A5E7B31EB3EF0CB2C00" : 0,
     "840BFB2A2F183FC5B01E0B21BB0D3D8EF3F6803F" : 0,
-    "D5FA66743146831DA2A20C0EAE4583EF424D9041" : 0
+    "D5FA66743146831DA2A20C0EAE4583EF424D9041" : 0,
+    "956D2B21DD155C49504BB67697A67F7C5351A353" : 0
   },
   "DVTSourceControlWorkspaceBlueprintIdentifierKey" : "16056085-48A1-4F20-BAB0-335120B8B6C5",
   "DVTSourceControlWorkspaceBlueprintWorkingCopyPathsKey" : {
     "9A760A1EBAC1979D46EF016B1BE84D98E53C16FF" : "Madness\/Carthage\/Checkouts\/Prelude\/",
     "7FCD68358D0D5A67F26F2A5E7B31EB3EF0CB2C00" : "Madness\/",
     "840BFB2A2F183FC5B01E0B21BB0D3D8EF3F6803F" : "Madness\/Carthage\/Checkouts\/Assertions\/",
-    "D5FA66743146831DA2A20C0EAE4583EF424D9041" : "Madness\/Carthage\/Checkouts\/Either\/"
+    "D5FA66743146831DA2A20C0EAE4583EF424D9041" : "Madness\/Carthage\/Checkouts\/Either\/",
```

Comment:
Can these references be 🔥d now?

Reply:
Does it matter?

Does anything matter?

Reply:
Ugh. This whole file should be ignored. I'll do that.